src/Accelerometer.cpp

```diff
@@ -1,78 +1,74 @@
 #include "Accelerometer.h"
 
-LIS331DLH::LIS331DLH(uint8_t addr)
-    : BaseIMU(addr) { }
+LIS331DLH::LIS331DLH(uint8_t slaveAddress)
+    : BaseIMU(slaveAddress) { }
 
 void LIS331DLH::begin(TwoWire& wire) {
-    // подключаемся к шине I²C
     _wire = &wire;
     _wire->begin();
-    // включаем координаты x, y, z
     _ctrlReg1 |= (1 << 0);
     _ctrlReg1 |= (1 << 1);
     _ctrlReg1 |= (1 << 2);
-    // включаем аксселерометр
     _ctrlReg1 |= (1 << 5);
-    // устанавливаем максимальное измеряемое ускорение в G
-    setRange(RANGE_2G);
     _writeByte(CTRL_REG1, _ctrlReg1);
+    setRange(RANGE_2G);
 }
 
 void LIS331DLH::setRange(uint8_t range) {
     switch (range) {
     case RANGE_2G: {
         _ctrlReg4 = ADR_FS_2;
-        _mult = RANGE_2G / 32767.0;
+        _scale = RANGE_2G / 32767.0;
```

Comment:
A strange divider. Should there really be 32767? Maybe 32768?

Reply:
This is data from older commit.
See the latest commit:
https://github.com/amperka/Troyka-IMU/pull/19/commits/66c569ac397a777327247eff8744b723ae126527